data/avro-schema/src/main/resources/avro/net/corda/data/flow/event/FlowEvent.avsc

```diff
@@ -11,7 +11,6 @@
       "name": "payload",
       "type": [
         "net.corda.data.flow.event.StartFlow",
-        "net.corda.data.flow.event.Wakeup",
```

Comment:
It's a remote edge case, but removing types like this could be a problem when upgrading from 5.0 to 5.1, need to check with @simon-johnson-r3, also worth adding an avro upgrade test to validate the change (see existing tests for an example)

Reply:
I think upgrade guarantees are just that you won't need to recompile your application. In particular I don't think we're guaranteeing that in flight flows will work across an upgrade, so any events of these types present in the system will need to be cleared before the upgrade can go ahead.